src/vm/types/pedersen_instance_def.zig

```diff
@@ -20,5 +27,124 @@ pub const PedersenInstanceDef = struct {
     /// The upper bound on the hash inputs.
     ///
     /// If None, the upper bound is 2^element_bits.
-    _hash_limit: std.math.big.int.Managed,
+    _hash_limit: ManagedBigInt,
+    /// The memory allocator. Can be needed for the deallocation of the VM resources.
+    allocator: Allocator,
+
+    pub fn default(allocator: Allocator) !Self {
```

Comment:
Please sync with @jobez regarding naming conventions and initializing default structs.

Reply:
hey! lets go with default -> initDefault

see https://github.com/ziglang/zig/blob/master/lib/std/enums.zig#L737 as prior art